vite.config.js

```diff
@@ -23,6 +23,8 @@ module.exports = defineConfig({
     },
   },
   build: {
+    sourcemap: 'inline',
```

Comment:
This doesn't seem to do much but I would like the errors to look better.

Will revert here and try to introduce a new PR that actually has an effect.